src/typedefs.js

```diff
@@ -293,4 +306,26 @@ const React = require('react');
  * @memberof typedefs
  */
 
+/**
+ * setAclPermissionModalProps is an object that stores the props for the setAclPermissionModal component.
+ *
+ * @exports setAclPermissionModalProps
+ * @typedef {object} setAclPermissionModalProps
+ * @property {boolean} showModal - Boolean for showing setAclPermissionModal.
+ * @property {React.Dispatch<React.SetStateAction<boolean>>} setShowModal
+ * - React set function for setting showModal state
+ * @property {(string) => void} handlePermissions
```

Comment:
The last prop for `SetAclPermissionModal` seems to be a string(?) called `documentName` and not the handler function `handlePermissions`.